.changeset/little-crabs-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+feat: add new `wrangler telemetry` commands to configure whether Wrangler collects usage telemetry
```

Comment:
Should we make it clear that these commands are currently hidden and will be exposed in a future release once telemetry collection is fully implemented?

Reply:
yep! also wondering if this is 'experimental' and thus a patch?

Reply:
Either way works for me 🤔

Reply:
d697b4da81c39e31e6d1379950fb32876bc3e895